src/EventStore.Client.Streams/EventStoreClient.Delete.cs

```diff
@@ -12,77 +12,65 @@ public partial class EventStoreClient {
 		/// </summary>
 		/// <param name="streamName">The name of the stream to delete.</param>
 		/// <param name="expectedRevision">The expected <see cref="StreamRevision"/> of the stream being deleted.</param>
+		/// <param name="deadline">The maximum time this operation is allowed to take.</param>
```

Comment:
This might imply something like 'it wont happen if it takes longer than it is allowed'.. Its hard to write a description for this that communicates that the call can still succeed on the server even if the client gets a timeout.. maybe just leave it blank?

or maybe `The maximum time to wait before terminating the call.` is sufficient